environments/india/public.yml

```diff
@@ -117,3 +117,6 @@ site_locations:
     try_files: "$uri $uri/index.html $uri/ =404"
     auth_basic: '"Restricted Content"'
     auth_basic_user_file: '/etc/nginx/.htpasswd_enikshay_private'
+
+commcare_cloud_root_user: ubuntu
+commcare_cloud_pem: ~/.ssh/id_rsa
```

Comment:
why did these get added? are they correct?

Reply:
Yeah this is right. These are also in `production/public.yml` (and `staging/public.yml`), and they together outline the "factory auth" for this environment. While on some environments the "factory auth" (the auth you should use to log into a brand new machine for the first time, usually to run `bootstrap-users`) is a username and a password, on AWS (and I'm sure some other providers) factory auth is done with the ssh key of the person who created the machine (logging in as the `ubuntu` user for Ubuntu AMIs). AWS lets you update a bunch of ssh public keys and then choose one to use when you create a new EC2 instance; our terraform code is set up to automatically select the key of person creating it.